app/controllers/application_controller.rb

```diff
@@ -1,8 +1,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery unless: -> { request.format.json? }
 
-  require 'authenticator'
-  require 'panoptes_client'
+  require 'panoptes_api'
```

Comment:
you shouldn't have to require these files if rails is autoloading from your lib dir, (you can modify which dirs are autoloaded).

Reply:
Files in `lib` aren't autoloaded, presumably to minimize eager_loading of vendor stuff. Files from `app/` are, though, so I suppose I could add an `app/lib` folder. Is that better than this line?